src/libs/data-utils.js

```diff
@@ -277,7 +297,22 @@ export const EntityUploader = ajaxCaller(class EntityUploader extends Component
           style: { color: colors.orange[0], fontWeight: 'bold', fontSize: 12, marginBottom: '0.5rem' }
         }, [isInvalid === 'file' ? 'Only .tsv files can be uploaded.' : 'File does not start with entity or membership definition.']),
         inputLabel('Selected File'),
-        (file && file.name) || div({ style: { color: colors.gray[2] } }, 'None'),
+        div({ style: { marginLeft: '0.5rem' } }, [
+          (file && file.name) || div({ style: { color: colors.gray[2] } }, 'None')
+        ]),
+        file && supportsFireCloudDataModel(newEntityType) && div([
+          h(LabeledCheckbox, {
+            checked: useFireCloudDataModel,
+            onChange: checked => this.setState({ useFireCloudDataModel: checked }),
+            disabled: !supportsFireCloudDataModel(newEntityType),
```

Comment:
nit: This seems unneeded since we skip rendering if it's false.

Reply:
True. That's left over from experimenting with always showing it regardless of the detected entity type. I'll check with Brad to see whether or not he wants that.

Reply:
I'd love to have it always visible, I feel like that would reduce confusion from people who are expecting it...

Reply:
Brad is preferring it as-is so I'll remove the unneeded `disabled` attribute. We can always revisit this if we change out minds.